k8s/apps/elasticsearch.yml

```
@@ -0,0 +1,47 @@
+    matchLabels:
+      app: mindjunkies-elasticsearch
+  template:
+    metadata:
+      labels:
+        app: mindjunkies-elasticsearch
+    spec:
+      containers:
+      - name: elasticsearch
+        image: docker.elastic.co/elasticsearch/elasticsearch:8.13.0
+        ports:
+        - containerPort: 9200
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 10 but found 8

(indentation)